modules/core/test/v2/integration/coins/eth.ts

```diff
@@ -17,6 +19,34 @@ describe('ETH:', function() {
     wallet = yield bitgo.coin('teth').wallets().getWallet({ id: TestBitGo.V2.TEST_ETH_WALLET_ID });
   }));
 
+  describe('Send Transaction', function() {
+    it('should send to multiple recipients in single transaction', async function() {
+      await bitgo.unlock( { otp: '0000000' });
+
+      const transaction = await wallet.sendMany({
+        recipients: [
+          {
+            amount: '1',
+            address: '0x431745b89e73230b3bc8a19e019194efb4b99efd',
+          },
+          {
+            amount: '5',
+            address: '0x431745b89e73230b3bc8a19e019194efb4b99efd',
+          },
+        ],
+        walletPassphrase: TestBitGo.V2.TEST_ETH_WALLET_PASSPHRASE,
+      });
+
+      should.exist(transaction);
+      transaction.should.have.property('transfer');
```

Comment:
Worth checking to make sure the entries are properly populated?

Reply:
Added check inside entries